src/components/UniversalNavbarExpanded/MobileNav/MobileNav.js

```diff
@@ -0,0 +1,174 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+
+// Reused assets from UniversalNavbar
+import FancyAnimatedLogo from '../../UniversalNavbar/FancyAnimatedLogo'
+import LogoWhite from '../../UniversalNavbar/assets/ethos-logo-white.js'
+import TransformingBurgerButton from '../../UniversalNavbar/TransformingBurgerButton/TransformingBurgerButton'
+
+// Parent component (UniversalNavbar) siblings
+import CtaButton from '../CtaButton'
+import NavLink from '../NavLink'
+
+// MobileNav siblings
+import AccordionNav from './AccordionNav'
+import LinkList from './LinkList'
+
+// Helpers
+import { isEnterKeyPress } from '../../../helpers/isEnterKeyPress'
+
+// Styles
+import styles from './MobileNav.module.scss'
+
+/**
+ * Extension of UniversalNavbar/TransformingBurgerButton, reused twice in the MobileNav
+ *
+ * @param {string} className - Extra top level class
+ * @param {boolean} visibleState - State for tracking visibility of MobileNav
+ * @param {function} clickHandler - Toggle function for visibleState
+ * @param {function} keyPressHandler - Toggle function for visibleState
+ *
+ * @return {JSX.Element}
+ */
+const BaseHamburger = ({
+  className,
+  visibleState,
+  clickHandler,
+  keyPressHandler,
+}) => {
+  return (
+    <div className={className}>
+      <TransformingBurgerButton
+        showMobileMenu={visibleState}
+        clickHandler={() => clickHandler()}
+        keyPressHandler={(e) => keyPressHandler(e)}
+      />
+    </div>
+  )
+}
+
+BaseHamburger.propTypes = {
+  className: PropTypes.string.isRequired,
+  visibleState: PropTypes.bool.isRequired,
+  clickHandler: PropTypes.func.isRequired,
+  keyPressHandler: PropTypes.func.isRequired,
+}
+
+/**
+ * Mobile navigation component for UniversalNavbar.
+ * Uses a full screen modal containing a dropdown accordion system and static link list.
+ *
+ * TODO Make the link list optional for better reusability
```

Comment:
Non blocking - if we ever want to use this without the links below the accordion.